api/src/main/java/org/pmiops/workbench/db/model/Cohort.java

```diff
@@ -85,7 +85,10 @@ public void setWorkspaceId(long workspaceId) {
     this.workspaceId = workspaceId;
   }
 
-  @Column(name = "criteria")
+  // This is actually a BLOB type. The default length limit in Spring Data for
```

Comment:
I think maybe we instead want to use the Lob annotation:

https://www.logicbig.com/tutorials/java-ee-tutorial/jpa/lob-annotation.html

We should check to see if this applies to other fields...

Reply:
Done. I saw `@Lob` suggested on a few threads, but I wasn't able to find any documentation as to what affect this would have on Spring Data JPA, so I didn't use it initially. It seems to work, so I guess it's fine. If you're familiar with this annotation and think it won't have any unintended side-effects, then I'll go with this.

There are no other "blob" types in db or db-cdr, so I guess this is the only one, unless there's another SQL type to worry about here.

Reply:
Try adding a test that inserts a cohort with large criteria?

Reply:
Never mind, I guess the BigQuery tests already do this. :)

Reply:
Yeah, I checked they were failing locally, and now they're passing.